packages/radix-ui-themes/src/components/dropdown-menu.tsx

```diff
@@ -300,7 +299,7 @@ const DropdownMenuSubContent = React.forwardRef<
   const { size, variant, color, highContrast } = React.useContext(DropdownMenuContentContext);
   const { className, children, container, forceMount, ...subContentProps } = extractProps(
     { size, variant, color, highContrast, ...props },
-    baseMenuContentPropDefs
```

Comment:
curious why we don't see 2 changes in that diff for dropdown menu too? Shouldn't there be one for the content part too?

Reply:
I think we were just random in this file, `baseMenuContentPropDefs` isn't used anywhere it shouldn't be anymore

<img width="512" alt="image" src="https://github.com/radix-ui/themes/assets/8441036/5fd1106e-53d8-4c95-bde0-4abcc94fdcfa">